hack/make-rules/verify.sh

```diff
@@ -45,12 +45,13 @@ if [[ ${EXCLUDE_TYPECHECK:-} =~ ^[yY]$ ]]; then
 fi
 
 
-# Exclude godep checks in certain cases, if they're running in a separate job.
+# Exclude vendor checks in certain cases, if they're running in a separate job.
 if [[ ${EXCLUDE_GODEP:-} =~ ^[yY]$ ]]; then
   EXCLUDED_PATTERNS+=(
     "verify-godeps.sh"             # runs in separate godeps job
-    "verify-staging-godeps.sh"     # runs in separate godeps job
-    "verify-godep-licenses.sh"     # runs in separate godeps job
+    "verify-godeps-licenses.sh"    # runs in separate godeps job
```

Comment:
why are we keeping this one?

Reply:
CI jobs reference it explicitly, and I want CI signal on this PR